Aptoide/aptoide-client-v8
# Feature/asv 2161 instabug

**What does this PR do?**

  Integrates InstaBug - ASV-2161

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] cm.aptoide.pt.AptoideApplication.java

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [ASV-2161](https://aptoide.atlassian.net/browse/ASV-2161)
   Open the app and shake it. You should see a InstaBug dialog.


**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2161](https://aptoide.atlassian.net/browse/ASV-2161)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.)  Yes, get keys from repo



**Code Review Checklist**
